cli/command/container/cp.go

```diff
@@ -34,11 +39,91 @@ const (
 type cpConfig struct {
 	followLink bool
 	copyUIDGID bool
+	quiet      bool
 	sourcePath string
 	destPath   string
 	container  string
 }
 
+//CopyReader is a Wrapper to content when it is a Reader
+type CopyReader struct {
+	io.Reader
+	total int64
+}
+
+//CopyReadCloser is a Wrapper to content when it is a ReadCloser
+type CopyReadCloser struct {
+	io.ReadCloser
+	total int64
+}
+
+var (
```

Comment:
These variables should be localized to where they are used instead of singletons.

Reply:
I did not understand. Some of these variables need to be used in different scopes (functions), so I ended up making them "global" to the cp.go package.

Reply:
The issue is, as I recall, that these values are being changed, which is undesirable.

Reply:
Hm, so I need move these variables to their respective functions ou could I make pointers to them?

Reply:
What do you think, @cpuguy83, if we move these fields to the types `CopyReader`, initialize them and in the `Read` function use them? So, with this change I make only one type to the `CopyReader` and `CopyReadCloser`.

Reply:
Hi @cpuguy83, how are you? I've made the changes. Now, only one type exists to perform the copy process and the variables are inside this type, because they are used during the `Read` method. Could you review?